packages/runtime-core/src/vnode.ts

```diff
@@ -215,7 +214,7 @@ export function createBlock(
     true /* isBlock: prevent a block from tracking itself */
   )
   // save current block children on the block vnode
-  vnode.dynamicChildren = currentBlock || EMPTY_ARR
+  vnode.dynamicChildren = currentBlock
```

Comment:
The current block is Array for stable fragment .
The current block is null (openBlock(true)) for unkeyed or keyed fragment.
So only check dynamicChildren if it is null when  patch or unmount.

Reply:
If it is correct. Maybe we can remove fragment flags in PatchFlags. Instead of add some comment to remind this.